neo/tools/debugger/DebuggerServer.cpp

```diff
@@ -197,11 +220,11 @@ bool rvDebuggerServer::ProcessMessages ( void )
 
 			case DBMSG_STEPOVER:
 				mBreakStepOver = true;
-				mBreakStepOverDepth = mBreakInterpreter->GetCallstackDepth ( );
-				mBreakStepOverFunc1 = mBreakInterpreter->GetCallstack()[mBreakInterpreter->GetCallstackDepth()].f;
-				if ( mBreakInterpreter->GetCallstackDepth() > 0 )
+				mBreakStepOverDepth = ((idGameEditExt*) gameEdit)->GetInterpreterCallStackDepth(mBreakInterpreter);
```

Comment:
can we run into this code (or any other code in this source file that uses `idGameEditExt`) if a (mod) game.dll that doesn't implement idGameEditExt is loaded?

Reply:
No, no code that uses the idGameEditExt can run if CheckBreakpoints() is not called.
If an debugger clients connects, then this will blow up spectacularly .
Maybe this should be prevented, i think i will start off with an hacky global like you proposed below.

Reply:
yeah, that's what I meant in https://github.com/dhewm/dhewm3/pull/381#discussion_r661338407 ;)

Reply:
yea i failed your point the first time, did see it the second time ;) thx for poining this out twice!